CMSgov/easi-app
# [EASI-2761] Add question about UI changes to request details (front and backend)

# EASI-2761

## Changes and Description

- Added backend migration, mutations, etc for whether or not an intake has UI changes
- Added the question to the request details form, updated tests, snapshots, etc.



## How to test this change
Run EASi locally, head to http://localhost:3000/system/making-a-request and create a system intake.

In the request details section, enter a value for the new question "Does your project involve any user interface component, or changes to an interface component?"
![image](https://user-images.githubusercontent.com/95702411/234747943-3ff17878-2b9a-4cb8-b796-83f45ef08a8f.png)

On the review page, confirm that your entry is shown as expected:
![image](https://user-images.githubusercontent.com/95702411/234748114-c5862e82-3038-4b03-9132-ac3b2774cb1a.png)

## PR Author Review Checklist


## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
